Nowe_Dane.md

```diff
@@ -0,0 +1,29 @@
+### Nowe Dane
+
+Dysponujemy dodatkowymi zbiorami uzupełniające nasze informacje. 
+Pierwsze trzy zawierają informacje o nagrodach za filmy.
+Warto zwrócić uwagę, że Oscary przyznawane są za produkcje kinowe, Emmy-telewizyjne, natomiast Złote Globy za oba typy produkcji.
+Dzięki wykorzystniu wszystkich zbiorów danych z nagrodami możemy uzyskać pełniejszy obraz. Pod odstatnim linkiem znajduje się baza TMBD. Znajdują się tam między innymi informacje o popularności filmów oraz ich budżetach.
+
+
+##### Oscary
+Zbiór danych pod poniższym linkiem zawiera o nominowanych oraz zdobywców Oscarów w latach 1927-2018.
```

Comment:
A jakieś uwagi o danych które pan przejrzał. Jakieś kolumny tam są ? Czy mamy pomysł co zrobić dla danych za lata 2019-present?

Reply:
Opisałem krótko te dane. W mojej ocenie to w jakich sposób użyć tych danych może być warto przedyskutować kiedy reszta zespołu się z nimi zapozna. Dane można prawdopodobnie rozszerzyć sięgając do źródła danych z kaggle ale nie mam pomysłu jak to zrobić.